javascripts/quiz.js

```diff
@@ -0,0 +1,54 @@
+"use strict";
+var insert = document.getElementById("insert");
+var editTarget;
+var descText = '';
+
+function populatePage (inventory) {
+  // we have a main div
+  // build one massive string to insert there
+  // Loop over the inventory and populate the page
+  var tempAddition = '';
+  for (let i = 0; i < inventory.length; i++){
+    if (i % 3 === 0){
+      tempAddition += '<div class="row">';
+    } 
+    tempAddition += '<div class="col-xs-4 ' + i + '" id="';
+    tempAddition += inventory[i].inventoryNum + i;
+    tempAddition += '"><div class="car-card col-xs-12"><div class="panel panel-default">';
+    tempAddition += '<div class="panel-heading"><h3 class="panel-title">';
```

Comment:
**fix layout claustrophobia**
added a div to my div in order to take advantage of Bootstrap's native margins